docs/www/get.html

```diff
@@ -31,7 +31,7 @@
   </div>
   <div class="col-xs-12 col-sm-4 text-center">
     <h3>Mac OSX</h3>
-    <img src="[% combust.static_url("/images/logos/mac.gif") %]" width="120" alt="Windows" style="margin:auto;"/>
+    <img src="[% combust.static_url("/images/logos/mac.gif") %]" width="120" alt="Mac OSX" style="margin:auto;"/>
```

Comment:
It should be macOS now.

https://en.wikipedia.org/wiki/Macintosh_operating_systems#Mac_OS_X.2FOS_X.2FmacOS